zegami_sdk/annotation.py

```diff
@@ -340,3 +340,53 @@ def base64_to_boolmask(b64_data):
                            np.array(img), dtype='uint8')
 
         return arr_int > 127
+
+
+class AnnotationBB(_Annotation):
+    """
+    An annotation comprising a bounding box and some metadata.
+
+    # TODO
+    To view the bounding boxes of an image, use bb.view().
+
+    Note: Providing imageset_id and image_index is not mandatory and can be
+    obtained automatically, but this is slow and can cause unnecessary
+    re-downloading of data.
+    """
+
+    TYPE = 'zc-boundingbox'
+    UPLOADABLE_DESCRIPTION = """
+        Bounding box annotation data includes the bounding box bounds,
+        a width and height, and score if generated
+        by a model (else None).
+    """
+
+    @classmethod
+    def create_uploadable(cls, bounds, class_id):
+        """
+        Creates a data package ready to be uploaded with a collection's
+        .upload_annotation().
+
+        Input 'bounds' is a dictionary of { height, width, x, y } for the
+        edges of the given bounding box.
+
+        Note: The output of this is NOT an annotation, it is used to upload
+        annotation data to Zegami, which when retrieved will form an
+        annotation.
+        """
+
+        data = {
+            'type': cls.TYPE,
+            'x': int(bounds['x']),
+            'y': int(bounds['y']),
+            'w': int(bounds['width']),
+            'h': int(bounds['height']),
```

Comment:
Would it be easier to have the user to provide `w`, `h`, or is it fairly standard to use `width` and `height`?

Reply:
I would say it's standard to use `width` and `height`: 
![image](https://user-images.githubusercontent.com/89839994/159944685-f1523563-3e2d-4e88-8967-dc6c1dd25e8b.png).

But might be easier to write just one letter...